resources/views/account/partials/details/title.blade.php

```diff
@@ -16,13 +16,13 @@
                     <x-rapidez-ct::title href="/account/wishlists"></x-rapidez-ct::title>
                     <div class="relative border rounded">
                         <x-rapidez::input class="w-80 border-0 pr-12" name="Edit title" type="text" v-model="editing.title" />
-                        <x-rapidez-mw::button.accent class="!absolute -translate-y-1/2 top-1/2 right-1  ml-4 flex items-center justify-center !p-0 w-9 h-9">
-                            <x-heroicon-o-pencil-square class="w-4 h-4 text-white" />
-                        </x-rapidez-mw::button.accent>
+                        <x-rapidez::button.secondary class="!absolute -translate-y-1/2 top-1/2 right-1  ml-4 flex items-center justify-center !p-0 size-9">
```

Comment:
absolute with important? is that necessary? is there a relative by default on the button?

Reply:
Correct see: https://github.com/rapidez/blade-components/blob/master/resources/views/components/button/button.blade.php